milatools/cli/commands.py

```diff
@@ -561,15 +578,42 @@ def code(
     command_path = shutil.which(command)
     if not command_path:
         raise CommandNotFoundError(command)
+    qualified_node_name = qualified(node_name)
+
+    # TODO: Find the path to this file on other platforms.
+    vscode_settings_json_path = Path.home() / (
+        "AppData\\Roaming\\Code\\User\\settings.json"
+        if (sys.platform == "win32" or running_inside_WSL)
+        else "~/.config/Code/User/settings.json"
+    )
+
+    if vscode_settings_json_path.exists():
+        print(T.bold(f"Found VSCode settings file at {vscode_settings_json_path}"))
+        set_remote_ssh_vscode_settings(
+            vscode_settings_json_path=vscode_settings_json_path,
+            timeout_seconds=60,
+            fully_qualified_node_name=qualified_node_name,
+        )
```

Comment:
I feel like this should be in `mila init` instead and that there should be a confirmation of the diff by the user.

Reply:
True!

Reply:
Removed this in fc61b941fdd5b5ed907b1f667c24a5dd1cd20c10